lib/train/file.rb

```diff
@@ -24,7 +24,7 @@ def sanitize_filename(_path)
     # interface methods: these fields should be implemented by every
     # backend File
     DATA_FIELDS = %w{
-      exist? mode owner group uid gid content mtime size selinux_label path
+      exist? mode owner group uid gid content content= mtime size selinux_label path
```

Comment:
I found that if you rename the method to be `set_content`, you can avoid the name issue, but then you encounter arity issues in that all of the other data fields are expected to be called with no arguments. If the new method is omitted from the DATA_FIELDS list - it is not really a data field, it is a setter - and instead treated as an abstract method, the tests pass.

Reply:
That was an easy fix. Thank you!